src/__tests__/Search.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Search from '../components/search';
+
+describe('search', () => {
+  const { asFragment } = render(<Search />);
+
+  it('renders correctly', () => {
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
It would be good to add a test here to check if the function is called when clicking the submit button. Could do this like so:

```
  it("button fires when clicked", async () => {
    render(<Search setSearchResults={jest.fn()} />);

    const button = screen.getByText("Go!");

    expect(button).toBeInTheDocument();
    fireEvent.click(button);
    expect(await validProps.setSearchResults).toHaveBeenCalledTimes(1);
  });
```